lib/spaceship/tunes/app_submission.rb

```diff
@@ -0,0 +1,155 @@
+      
+      # @return (Boolean) Export Compliance - Contains third-party cryptography
+      attr_accessor :export_compliance_contains_third_party_cryptography
+      
+      # @return (Boolean) Export Compliance - Is exempt
+      attr_accessor :export_compliance_is_exempt
+      
+      # @return (Boolean) Export Compliance - Uses encryption
+      attr_accessor :export_compliance_uses_encryption
+
+      # @return (String) Export Compliance - App type
+      attr_accessor :export_compliance_app_type
+      
+      # @return (Boolean) Export Compliance - Encryption Updated
+      attr_accessor :export_compliance_encryption_updated
+      
+      # @return (Boolean) Export Compliance - Compliance Required
+      attr_accessor :export_compliance_compliance_required
+      
+      # @return (String) Export Compliance - Platform
+      attr_accessor :export_compliance_platform
+
+      attr_mapping({
+
+        # Ad ID Info Section
+        'adIdInfo.limitsTracking.value' => :add_id_info_limits_tracking,
+        'adIdInfo.servesAds.value' => :add_id_info_serves_ads,
+        'adIdInfo.tracksAction.value' => :add_id_info_tracks_action,
+        'adIdInfo.tracksInstall.value' => :add_id_info_tracks_install,
+        'adIdInfo.usesIdfa.value' => :add_id_info_uses_idfa,
+        
+        # Content Rights Section
+        'contentRights.containsThirdPartyContent.value' => :content_rights_contains_third_party_content,
+        'contentRights.hasRights.value' => :content_rights_has_rights,
+        
+        # Export Compliance Section
+        'exportCompliance.availableOnFrenchStore.value' => :export_compliance_available_on_french_store,
+        'exportCompliance.ccatFile.value' => :export_compliance_ccat_file,
+        'exportCompliance.containsProprietaryCryptography.value' => :export_compliance_contains_proprietary_cryptography,
+        'exportCompliance.containsThirdPartyCryptography.value' => :export_compliance_contains_third_party_cryptography,
+        'exportCompliance.isExempt.value' => :export_compliance_is_exempt,
+        'exportCompliance.usesEncryption.value' => :export_compliance_uses_encryption,
+        'exportCompliance.appType' => :export_compliance_app_type,
+        'exportCompliance.encryptionUpdated' => :export_compliance_encryption_updated,
+        'exportCompliance.exportComplianceRequired' => :export_compliance_compliance_required,
+        'exportCompliance.platform' => :export_compliance_platform
+      })
+
+      class << self
+        # Create a new object based on a hash.
+        # This is used to create a new object based on the server response.
+        def factory(attrs)
+          orig = attrs.dup
+          
+          # fill content rights section if iTC returns nil
+          if attrs["contentRights"].nil?
+            attrs.merge!("contentRights" => {
+              "containsThirdPartyContent" => {
+                "errorKeys" => nil,
+                "isEditable" => false,
```

Comment:
You don't have to send `isEditable`, etc. to iTunes Connect, it's enough to only have a `value`.